static/bidder-info/tradplus.yaml

```diff
@@ -0,0 +1,15 @@
+endpoint: "https://{{.ZoneID}}adx.tradplusad.com/{{.AccountID}}/pserver"
+
+maintainer:
+  email: "tpxcontact@tradplus.com"
```

Comment:
@tradplus We have sent a verification email on above address. Please respond to the mail with 'Received' to complete the process. Thank you!

Reply:
done.